plugins/package-managers/swiftpm/src/funTest/assets/projects/synthetic/spm-lib-no-lockfile/Package.resolved

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
Hmm, the rationale is "to avoid the need for constantly updating the expected results", but won't that still be the case for the copied `spm-lib` project without the lockfile?

Reply:
For the copied project, the test only verifies that the analyzer rejects analysis because `allowDynamicVersions = false`.